apps/web/app/(auth)/invite/page.tsx

```diff
@@ -11,21 +10,20 @@ import {
   RightAccountContent,
 } from "./components/InviteContentComponents";
 import { env } from "@formbricks/lib/env.mjs";
+import { deleteInvite, getInvite } from "@formbricks/lib/invite/service";
+import { createMembership } from "@formbricks/lib/membership/service";
 
 export default async function JoinTeam({ searchParams }) {
   const currentUser = await getServerSession(authOptions);
 
   try {
-    const { inviteId, email } = await verifyInviteToken(searchParams.token);
+    const { inviteId, email } = verifyInviteToken(searchParams.token);
 
-    const invite = await prisma?.invite.findUnique({
-      where: { id: inviteId },
-      include: { creator: true },
-    });
+    const invite = await getInvite(inviteId);
 
-    const isExpired = (i) => new Date(i.expiresAt) < new Date();
+    const isExpired = (expiresAt: Date) => new Date(expiresAt) < new Date();
```

Comment:
I think we need `isExpired` as function here but could also directly calculate the value?
```ts
const isInviteExpired = new Date(invite.expiresAt) < new Date();
```